freshdesk/v2/models.py

```diff
@@ -123,5 +123,37 @@ def __str__(self):
         return self.name
 
     def __repr__(self):
-        return '<Role \'{}\'>'.format(self.name)    
+        return '<Role \'{}\'>'.format(self.name)
 
+class Category(FreshdeskModel):
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        return '<Category \'{}\'>'.format(self.name)        
+
+class Folder(FreshdeskModel):
+    def __str__(self):
+        return self.name
+
+    def __repr__(self):
+        return '<Folder \'{}\'>'.format(self.name)
+
+    #@property
+    #def visibility(self):
+    #    _s = {1: 'note', 2: 'twitter', 3: 'survey', 4: 'facebook'}
+    #    return _s[self._visibility]
+
+ 
+
+class Article(FreshdeskModel):
+    def __str__(self):
+        return self.title
+
+    def __repr__(self):
+        return '<Article \'{}\'>'.format(self.title)
+
+    #@property
```

Comment:
This too :-)

Reply:
I`ll create a Python2.7 environment to test and fix =)

Reply:
Great, thanks. I'll leave this out of current release but can do another point release when it's ready.